src/applications/static-pages/facilities/buildVamcOperatingStatusProps.js

```diff
@@ -0,0 +1,30 @@
+export const buildVamcOperatingStatusProps = attrs => {
```

Comment:
Naming on this is confusing.  It is not VamcOperatingstatus.     If anything it is supplemental status. ... but going even bigger it is alert expandable